doc/manuals/installation_and_administration_manual/streaming/annex_c_deploy_storm_cluster.md

```diff
@@ -0,0 +1,191 @@
+#<a name="top"></a>Annex C: Deploy Apache Storm Cluster
+
+Content:<br>
+
+* [Prepare Operating System](#section1)
+* [Deploy Storm](#section2)
+
+In order to complete Sinfonier installation it's necessary to deploy an Apache Storm cluster. This reference is based on CentOS 7 operating system and some specific software versions.
+
+##<a name="section1"></a>Prepare Operating System
+
+This guide use [CentOS 7](https://www.centos.org/) as based system where deploy Storm.
+
+**Add repositories**
+
+    https://github.com/CentOS/sig-atomic-buildscripts
+
+**Update your system**
+
+    $ sudo yum update
+
+**Install Java 7 SDK**
+
+    http://www.oracle.com/technetwork/es/java/javase/downloads/jdk7-downloads-1880260.html
```

Comment:
I would add something like "Please, follow [these guidelines](http://...) in order to install Java 7 SDK".